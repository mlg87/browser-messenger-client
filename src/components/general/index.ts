export { default as ChatBox } from './ChatBox/ChatBox';
export { default as UserCard } from './UserCard/UserCard';
export { default as UserList } from './UserList/UserList';
