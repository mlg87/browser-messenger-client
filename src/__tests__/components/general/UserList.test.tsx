import { render } from '@testing-library/react';
import { Provider } from 'mobx-react';
import React from 'react';
import { UserList } from '../../../components/general';
import appState from '../../../stores';

const setup = () => render(
    <Provider {...appState}>
        <UserList />
    </Provider>
);

describe('UserList', () => {

    it('renders', () => {

        const { container } = setup();
        expect(container.firstChild).toMatchSnapshot();

    });

});
