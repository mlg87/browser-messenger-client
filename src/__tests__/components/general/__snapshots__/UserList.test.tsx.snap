// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserList renders 1`] = `
<div
  class="container"
>
  <header>
    <h2
      class="sectionTitle"
    >
      users
    </h2>
    <i>
      select another user to start chatting
    </i>
  </header>
  <div>
    loading...
  </div>
</div>
`;
